pkg/runner/command_test.go

```diff
@@ -113,5 +116,55 @@ func TestAddmask(t *testing.T) {
 	handler := rc.commandHandler(loggerCtx)
 	handler("::add-mask::my-secret-value\n")
 
+	a.Equal("  \U00002699  ***", hook.LastEntry().Message)
 	a.NotEqual("  \U00002699  *my-secret-value", hook.LastEntry().Message)
 }
+
+// based on https://stackoverflow.com/a/10476304
+func captureOutput(t *testing.T, f func()) string {
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	f()
+
+	outC := make(chan string)
+
+	go func() {
+		var buf bytes.Buffer
+		_, err := io.Copy(&buf, r)
+		if err != nil {
+			a := assert.New(t)
+			a.Fail("io.Copy failed")
+		}
+		outC <- buf.String()
+	}()
+
+	w.Close()
+	os.Stdout = old
+	out := <-outC
+
+	return out
+}
+
+func TestAddmaskUsemask(t *testing.T) {
```

Comment:
```suggestion
func TestAddMaskUseMask(t *testing.T) {
```

Nitpick. How ever I do not understand the `Usemask` part?

Reply:
`TestAddmask` uses the same pattern, would like to keep them the same